models/MLmodels/ModelsRunner.py

```diff
@@ -13,21 +13,23 @@
 
 class ModelsRunner:
 
-    models_dict = {
-        'Lasso': Lasso(),
-        'Ridge': Ridge(),
-        'ElasticNet': ElasticNet(),
-        'DecisionTree': DecisionTree(),
-        'RandomForest': RandomForest(),
-        'NeuralNetwork': NeuralNetwork()
-    }
-
-    def __init__(self, path=None):
+    def __init__(self, path=None, data_sample=None):
         if path is not None:
```

Comment:
tip: this can be written as "if not path", in python "None" equivalates to "False" in conditions